modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -301,7 +301,7 @@
 
 // Responsive mode
 @mixin hide-normal-tabs-($device) {
-	.e-n-tabs-#{$device} {
+	.elementor-widget-n-tabs.e-n-tabs-#{$device} {
```

Comment:
I have made this more specific, because the previous styling was overruling our new styling.